cl_sii/rtc/xml_utils.py

```diff
@@ -29,10 +30,10 @@ def _get_signature(self, root: Any) -> object:
                 f'Only XML element {self.AEC_XML_ELEMENT_TAG!r} is supported. Found: {root.tag!r}',
             )
 
-        if root.tag == signxml.ds_tag("Signature"):
+        if root.tag == signxml.util.ds_tag("Signature"):
             return root
         else:
-            return self._find(root, "Signature", anywhere=False)
+            return self._find(root, "Signature")
```

Comment:
This may be a breaking change. Verify that and update dependency constraints if necessary: https://github.com/fyntex/lib-cl-sii-python/blob/b669efd2f47dad726379e763495bd7c25a1f5e95/setup.py#L32

Reply:
Fixed in https://github.com/fyntex/lib-cl-sii-python/pull/462.